tpv13/parameters.par

```diff
@@ -34,6 +36,7 @@ printtimeinterval_sec = 1.0                    ! Time interval at which output w
 OutputMask = 1 1 1 1 1 1 1 1 1 1 1             ! turn on and off fault outputs
 refinement_strategy = 2
 refinement = 1
+!vtkorder = 3
```

Comment:
Commented out?

Reply:
yes, because I decided not to use, because this would require updating the paraview example below.